features/step_definitions/logging.rb

```diff
@@ -286,7 +286,6 @@
 
 Given /^I delete the clusterlogging instance$/ do
   ensure_admin_tagged
-  ensure_destructive_tagged
```

Comment:
@QiaolingTang why remove the `destructive` tags?

Reply:
@pruan-rh I thought deleting the clusterlogging instance wouldn't affect the OCP cluster. But since deleting the CL instance could affect other logging scenarios, I'll add it back.  Thanks.